src/ModelClient.ts

```diff
@@ -24,6 +24,9 @@ export class ModelClient {
         this._apiHost = apiHost || '';
         this._fetchPromises = {};
     }
+    get apiHost(): string | null {
```

Comment:
Please provide a Doc

Reply:
oks

Reply:
`aemedit`

Reply:
1. in the constructor you have `this._apiHost = apiHost || '';` - i think `this._apiHost = apiHost || null;` would be better
2. in `fetch` reject promise in case host is not provided